chapters/angulardart/index.md

```diff
@@ -196,7 +194,7 @@ The functional capabilities define what AngularDart does. AngularDart is a port
 We derived the following functions in AngularDart:
 
 | Function | In component |
-|----------|--------|
+|----------------------------------------------------|------------|
```

Comment:
I sort of reverse engineered this, but I don't remember where I read about it. Probably inspired by
http://pandoc.org/README.html#tables

But indeed, when creating pdf or epub, pandoc needs to select a column width. It uses various heuristics, which can be influenced by the actual column widths in number of characters in headers like these.

This is indeed ignored by GFM / Jekyll.